frontend/src/components/GoalForm/index.js

```diff
@@ -587,10 +587,23 @@ export default function GoalForm({
 
       const updatedGoals = await createOrUpdateGoals(goals);
 
-      const updatedObjectives = updatedGoals && updatedGoals.length > 0
-        && updatedGoals[0] && updatedGoals[0].objectives && updatedGoals[0].objectives.length > 0
-        ? [...updatedGoals[0].objectives]
-        : [];
+      const newCreatedGoals = updatedGoals.filter((goal) => {
```

Comment:
This seems odd and I wonder if it can be written better. Is there chance here if the existing goal isn't found it will return unidentified?

Reply:
Also probably want to add a comment.

Reply:
@AdamAdHocTeam I was able to remove this